core-tests/jvm/src/test/scala/zio/SerializableSpec.scala

```diff
@@ -74,8 +74,8 @@ object SerializableSpec extends ZIOBaseSpec {
       }
     },
     testM("Duration is serializable") {
-      import zio.duration.Duration
-      val duration = Duration.fromNanos(1)
+      import zio.duration._
+      val duration = 1.nanos
```

Comment:
We should be able to keep `Duration.fromNanos` just for backward compatibility reasons.

Reply:
Plus other ones that existed, like `fromMillis`.

Reply:
It's still there but there are a few places where I used the new .nanos instead. Should I revert it in those places?

Reply:
I will revert the change to .nanos to make it easier to track changes.

Reply:
Yes, if you can do it without any changes outside `zio.duration._`, then it's a great sign users won't have any pains, either!